content/en/docs/reference/networking/virtual-ips.md

```diff
@@ -488,6 +488,63 @@ route to ready node-local endpoints. If the traffic policy is `Local` and there
 are no node-local endpoints, the kube-proxy does not forward any traffic for the
 relevant Service.
 
```

Comment:
I wonder if this mostly belongs under the heading `### Traffic to terminating endpoints`. (my guess is: yes it does; you could leave in a one sentence hyperlink to the moved text)

and somewhere you ought to add:

```markdown

{{< feature-state feature_gate_name="KubeProxyDrainingTerminatingNodes" >}}

```

Reply:
> I wonder if this mostly belongs under the heading ### Traffic to terminating endpoints

Not quite, what I've added deals with "traffic to terminating nodes"

> and somewhere you ought to add:

What does that do? The feature is beta in 1.30 so enabled by default.

Reply:
The feature state shortcode tells people they need a particular Kubernetes version to use it - see https://kubernetes.io/docs/contribute/new-content/new-features/#ready-for-review-feature-gates